chaoscenter/graphql/server/pkg/probe/utils/utils.go

```diff
@@ -347,6 +347,12 @@ func GenerateProbeManifest(probe *model.Probe, mode model.Mode) (string, error)
 			_probe.RunProperties.StopOnFailure = *probe.KubernetesCMDProperties.StopOnFailure
 		}
 
+		if probe.KubernetesCMDProperties.Source != nil {
+			var source v1alpha1.SourceDetails
+			_ = json.Unmarshal([]byte(*probe.KubernetesCMDProperties.Source), &source)
```

Comment:
We should log the err if any on this unmarshal

Reply:
Please also verify if you are able to run a CMD probe without providing any source input at all

Reply:
> We should log the err if any on this unmarshal

I have added a warn log (without return the error) when any fail ocurr when unmarshalling the source part of the probe. Thus, the source property of struct CmdProbeInputs will remain nil and in practice the source part of the probe will be ignored.

> Please also verify if you are able to run a CMD probe without providing any source input at all

When we doesn't provide any source the evaluation of the probe works as expected:

<img width="1943" alt="Screenshot 2024-10-11 at 11 19 30" src="https://github.com/user-attachments/assets/ac513313-0ab7-4b4b-894e-236108fd1b53">